src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-eks-add-on.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: New Relic Kubernetes Operator from AWS Marketplace
+tags:
+  - Infrastructure
+  - Install the EKS Add-on
+  - NewRelic Operator
+  - AWS EKS Add-on
+metaDescription: New Relic's EKS Add For EKS Monitoring.
+---
+
+import eksAddOnTab from 'images/EKS-Add-On-Screenshot-1.webp'
+
+import eksAddOnMarketplace from 'images/EKS-Add-On-Screenshot-2.webp'
+
+import eksAddOnMarketplaceConfig from 'images/EKS-Add-On-Screenshot-3.webp'
+
+<CollapserGroup>
+  <Collapser
+    id="what-is-extension"
+    title="What is an AWS EKS?"
+  >
+    Amazon Elastic Kubernetes Service (Amazon EKS) is a managed Kubernetes service that makes it easy for you to run Kubernetes on AWS and on-premises. Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications.
+  </Collapser>
+</CollapserGroup>
+
+## This document shows the steps required to use EKS AddOn from AWS Marketplace.
+#### **NOTE : This addon is currently supported only on X86 instance types and the support for ARM64 architecture type is coming soon**
```

Comment:
Should be "This add-on is currently only supported on x86 instance types, with support for ARM64 coming soon

Reply:
@glennthomas1, addressed, I made the suggested change.